medusa/indexers/tmdb/tmdb.py

```diff
@@ -190,25 +192,25 @@ def search(self, series):
 
         return OrderedDict({'series': mapped_results})['series']
 
-    def _get_shows_countries(self, tmdb_id):
-        """Retrieve show's countries from TMDB.
+    def get_show_country_codes(self, tmdb_id):
+        """Retrieve show's 2 letter country codes from TMDB.
 
-        :param tmdb_id: The shows tmdb id
-        :return: A string with the show's countries
+        :param tmdb_id: The show's tmdb id
+        :return: A list with the show's country codes
         """
         show_info = self._get_show_by_id(tmdb_id)['series']
 
-        return show_info.get('origin_country', '')
+        if show_info and show_info.get('origin_country'):
+            return show_info['origin_country'].split('|')
 
     def _get_show_by_id(self, tmdb_id, request_language='en'):  # pylint: disable=unused-argument
-        """Retrieve tmdb show information by tmdb id, or if no tmdb id provided by passed external id.
```

Comment:
@p0psicles 
I removed this because the ```tmdb_id``` isn't even optional and it was just a bit confusing.